tests/test_bench.py

```diff
@@ -0,0 +1,23 @@
+import os
+from kubernetes import client, config
+os.sys.path.append(os.path.dirname(os.getcwd()))
+
+from oc import pod, pod_exec
+from tasks import radosbench
+
+
+def run():
+    config.load_kube_config()
+    v1 = client.CoreV1Api()
+    ret = v1.list_pod_for_all_namespaces(watch=False,
+                                        label_selector='app=rook-ceph-tools')
+    for i in ret.items:
+        namespace = i.metadata.namespace
+        name = i.metadata.name
+        break
+
+    po = pod.Pod(name, namespace, roles=['client'])
+    radosbench.run(ceph_pods=[po],
```

Comment:
I'll be happy to see all of us aligns on the parenthesis code guideline:
foo(
    bar()
)

But in this case, it can fit 1 line
radosbench.run(ceph_pods=[po], config={'time': 10, })

Or 

radosbench.run(
    ceph_pods=[po], config={'time': 10, }
)

Reply:
We can use autopep8, my atom editor does that due to my plugin and we can have uniform standards, "autopep8 -a -a filename.py" https://pypi.org/project/autopep8/

Reply:
Done using autopep8

Reply:
Done. Used new coding guidelines